sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_web_referrer_v1/query.sql

```diff
@@ -0,0 +1,104 @@
+WITH views_territory AS (
+  SELECT
+    DATE(web_referrer.`date`) AS `date`,
+    web_referrer.app_id,
+    web_referrer.territory,
+    web_referrer.web_referrer,
+    COALESCE(campaign.campaign, "MISSING") AS campaign,
+    SUM(COALESCE(campaign.impressions, web_referrer.impressions)) AS impressions,
+    SUM(COALESCE(campaign.impressions_unique_device, web_referrer.impressions_unique_device)) AS impressions_unique_device,
+    SUM(COALESCE(campaign.page_views, web_referrer.page_views)) AS page_views,
+    SUM(COALESCE(campaign.page_views_unique_device, web_referrer.page_views_unique_device)) AS page_views_unique_device,
+  FROM
+    `moz-fx-data-bq-fivetran.firefox_app_store.app_store_territory_web_referrer_report` AS web_referrer
+  LEFT JOIN
+    `moz-fx-data-bq-fivetran.firefox_app_store.app_store_territory_campaign_report` AS campaign
+    ON web_referrer.`date` = campaign.`date`
+      AND web_referrer.app_id = campaign.app_id
+      AND web_referrer.territory = campaign.territory
+      AND web_referrer.web_referrer = "mozilla.org"
```

Comment:
This limits the results in this CTE to just `mozilla.org` web referrers

Reply:
The entries from the web_referrer should still be there since it's a left join? It's only the campaign entries which match should be included?